io.github.denzilferreira.asteroid_shooter.json

```diff
@@ -39,7 +39,7 @@
           "type": "git",
           "url": "https://github.com/denzilferreira/asteroid_shooter.git",
           "tag": "v1.0.0",
-          "commit": "c1724ba407d09006d2598809116970252bf1c95b"
+          "commit": "39d37ecad77812a12a68ee552fa13beeb86dec7a"
```

Comment:
retagging is bad.

Reply:
Should I make a new release with a new tag?... v1.0.0 is not really released yet...

Reply:
fair enough.